include/z64player.h

```diff
@@ -686,9 +686,9 @@ typedef struct Player {
     /* 0x0834 */ s16        unk_834;
     /* 0x0836 */ s8         unk_836;
     /* 0x0837 */ u8         unk_837;
-    /* 0x0838 */ f32        linearVelocity;
-    /* 0x083C */ s16        currentYaw;
-    /* 0x083E */ s16        targetYaw;
+    /* 0x0838 */ f32        speedXZ;
+    /* 0x083C */ s16        yaw;
+    /* 0x083E */ s16        zTargetYaw;
```

Comment:
```suggestion
    /* 0x0838 */ f32        speedXZ; // Main parameter for horizontal speed. Controls `Actor.speed`
    /* 0x083C */ s16        yaw; // Related to yaw (current or target)
    /* 0x083E */ s16        zTargetYaw; // Related to the player yaw when using Z to target an actor or a wall
```
?

Reply:
I forgot, for `zTargetYaw` : "(also boomerang related)" ?

Reply:
boomerang was just an example of an interaction with target i gave to show you that its related to targeting.
when you throw the boomerang, you enter a z target mode even tho you arent holding Z

Reply:
but yea I can add comments, tho I will prob rephrase

Reply:
I added some comments. I made the z target one fairly vague, at least until we know all of the details